contrib/nydusify/cmd/nydusify.go

```diff
@@ -397,6 +398,58 @@ func main() {
 				return checker.Check(context.Background())
 			},
 		},
+		{
+			Name:  "view",
```

Comment:
Maybe we can add an alias `Aliases: []string{"mount"},`.

Reply:
OK!